packages/main/src/OpenedPopupsRegistry.js

```diff
@@ -0,0 +1,27 @@
+import { isEscape } from "@ui5/webcomponents-base/dist/events/PseudoEvents.js";
+
+let registry = [];
```

Comment:
new Set() will be easier, you can add/delete instances directly (no need to filter)

Reply:
Set does not guarantee the order of pushed elements, therefore I decided to go for the list implementation